lib/beaker-puppet/install_utils/aio_defaults.rb

```diff
@@ -15,15 +15,21 @@ module AIODefaults
             'distmoduledir'         => '/etc/puppetlabs/code/modules',
             'sitemoduledir'         => '/opt/puppetlabs/puppet/modules',
           },
-          'windows' => { #windows
+          # sitemoduledir not included Windows (check PUP-4049 for more info).
+          #
+          # Paths to the puppet's vendored ruby installation on Windows were
+          # updated in Puppet 6 to more closely match those of *nix agents.
+          # These path values include both the older (puppet <= 5) paths (which
+          # include sys/ruby) and the newer versions, which have no custom ruby
+          # directory
+          'windows' => { # windows with cygwin
             'puppetbindir'      => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/bin',
-            'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
+            'privatebindir'     => '/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/puppet/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/puppet/bin:/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin',
             'distmoduledir'     => '`cygpath -smF 35`/PuppetLabs/code/modules',
-            # sitemoduledir not included (check PUP-4049 for more info)
           },
-          'pwindows' => { #pure windows
+          'pswindows' => { # pure windows
```

Comment:
I kept seeing `'pswindows'` instead of `'pwindows'` elsewhere, so I've updated this to match -- please let me know if I'm wrong here